.github/scripts/get-release-date.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+RELEASE_DATE=$(curl -H "Accept: application/vnd.github.v3+json" -H "Authorization: token $GITHUB_TOKEN" "https://api.github.com/repos/$GITHUB_REPOSITORY/releases/tags/$INPUT_RELEASE" | jq -r '.published_at')
+echo "RELEASE_DATE=$RELEASE_DATE" >> $GITHUB_ENV
```

Comment:
I suggest avoiding using Env vars.
I think you should use GitHub output vars.
Here is a small implementation example.
https://github.com/davseve/github-output-var

Reply:
Since we broke it into separate files, now we can't use output vars.

Reply:
@mark-elementor I agree with David output vars are more explicit in their use. So less space for unintended side effects when we maintain a script longer term. You can use it in your bash directly. 

echo "RELEASE_DATE=$RELEASE_DATE" >> $GITHUB_OUTPUT

Reply:
Thanks for the discussion on this Mark. We're abandoning this for now because extracting scripts from the `yml` makes the use of output variables inside the scripts less readable. When the output variables are injected from `yml` they come in as arguments. So it results in $1 $2 instead of named variables in the scripts which makes code harder to understand.